constants/errors.go

```diff
@@ -31,4 +31,8 @@ var (
 	ErrorFileTooLarge = errors.New("file too large")
 	// ErrorFileNotExist returned while file is not found.
 	ErrorFileNotExist = errors.New("file not exist")
+	// ErrorFileSizeInvalid returned while get file size not valid.
+	ErrorFileSizeInvalid = errors.New("file size invalid")
```

Comment:
`Seek` will failed for `size invalid`? For example?

Reply:
> `Seek` will failed for `size invalid`? For example?

Just wrap the `Seek` throw-out err